recipes/PReFerSim/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: prefersim
+  version: '1.0'
+
+source:
+  url: https://github.com/LohmuellerLab/PReFerSim
```

Comment:
You need to tag a release

Reply:
I think I solved this. Just tagged a release.

The text files contain some examples to run the code and are included in the release. They can be helpful for the user to run their analysis. I would like to keep them in the release folder.

Thanks for your help!